### ISSUE TITLE
Application keeps asking for Strava authorisation

### ISSUE THREAD

**Issue by jaroslawhartman** [MathBunny/strava-wind-analysis#86]
Hi, 

This seems like an issue: after initial authorisation and loading list of my segments, when trying to select one of them, the app jumps again to Strava authorisation screen.

Tried on Mac OS, in Chrome and Safari - same behaviour. If important - I'm logging to Strava using email/password.

You can see this problem in [this recording.](https://youtu.be/NSt4N7XzJTI) 

Best regards,
Jarek

**Comment by MathBunny** [comment]
Thanks for the report, I'll look into it

**Comment by MathBunny** [comment]
Seems like a repeat of #83, but I believe I fixed it hopefully once and for all (well, at least until the Redis becomes too big to fit in RAM). I fixed the issue by running `sudo apt-get autoremove` after realizing that I ran out of disk space on my EC2 again, despite clearing the Redis cache just last week. 

Turns out, there was ~2.8GB of Linux boot images stored on the server that weren't needed! 😄 

```
ubuntu@ip-172-31-33-240:/usr/src$ sudo apt-get autoremove
Reading package lists... Done
Building dependency tree       
Reading state information... Done
The following packages will be REMOVED:
  linux-aws-headers-4.4.0-1022 linux-aws-headers-4.4.0-1028 linux-aws-headers-4.4.0-1030 linux-aws-headers-4.4.0-1031
  linux-aws-headers-4.4.0-1032 linux-aws-headers-4.4.0-1035 linux-aws-headers-4.4.0-1038 linux-aws-headers-4.4.0-1039
  linux-aws-headers-4.4.0-1041 linux-aws-headers-4.4.0-1043 linux-aws-headers-4.4.0-1044 linux-aws-headers-4.4.0-1047
  linux-aws-headers-4.4.0-1048 linux-aws-headers-4.4.0-1049 linux-aws-headers-4.4.0-1052 linux-aws-headers-4.4.0-1054
  linux-aws-headers-4.4.0-1057 linux-aws-headers-4.4.0-1060 linux-aws-headers-4.4.0-1061 linux-aws-headers-4.4.0-1062
  linux-aws-headers-4.4.0-1065 linux-headers-4.4.0-1022-aws linux-headers-4.4.0-1028-aws linux-headers-4.4.0-1030-aws
  linux-headers-4.4.0-1031-aws linux-headers-4.4.0-1032-aws linux-headers-4.4.0-1035-aws linux-headers-4.4.0-1038-aws
  linux-headers-4.4.0-1039-aws linux-headers-4.4.0-1041-aws linux-headers-4.4.0-1043-aws linux-headers-4.4.0-1044-aws
  linux-headers-4.4.0-1047-aws linux-headers-4.4.0-1048-aws linux-headers-4.4.0-1049-aws linux-headers-4.4.0-1052-aws
  linux-headers-4.4.0-1054-aws linux-headers-4.4.0-1057-aws linux-headers-4.4.0-1060-aws linux-headers-4.4.0-1061-aws
  linux-headers-4.4.0-1062-aws linux-headers-4.4.0-1065-aws linux-image-4.4.0-1022-aws linux-image-4.4.0-1028-aws
  linux-image-4.4.0-1030-aws linux-image-4.4.0-1031-aws linux-image-4.4.0-1032-aws linux-image-4.4.0-1035-aws
  linux-image-4.4.0-1038-aws linux-image-4.4.0-1039-aws linux-image-4.4.0-1041-aws linux-image-4.4.0-1043-aws
  linux-image-4.4.0-1044-aws linux-image-4.4.0-1047-aws linux-image-4.4.0-1048-aws linux-image-4.4.0-1049-aws
  linux-image-4.4.0-1052-aws linux-image-4.4.0-1054-aws linux-image-4.4.0-1057-aws linux-image-4.4.0-1060-aws
  linux-image-4.4.0-1061-aws linux-image-4.4.0-1062-aws linux-image-4.4.0-1065-aws
0 upgraded, 0 newly installed, 63 to remove and 115 not upgraded.
After this operation, 2,800 MB disk space will be freed.
```

**Comment by MathBunny** [comment]
Fixed the issue, seems related to #83 - but this time it should be fixed for good (until the Redis cache exceeds RAM at least). I was surprised that this turned out again, because just last weak I cleared out the Redis cache for a similar issue. Turns out, the Linux AWS headers were taking up too much space, so nothing could be written to the EC2 instance. However, running `apt-get autoremove` cleared 2.8GB of unneeded headers.

```
ubuntu@ip-172-31-33-240:/usr/src$ sudo apt-get autoremove
Reading package lists... Done
Building dependency tree       
Reading state information... Done
The following packages will be REMOVED:
  linux-aws-headers-4.4.0-1022 linux-aws-headers-4.4.0-1028 linux-aws-headers-4.4.0-1030 linux-aws-headers-4.4.0-1031
  linux-aws-headers-4.4.0-1032 linux-aws-headers-4.4.0-1035 linux-aws-headers-4.4.0-1038 linux-aws-headers-4.4.0-1039
  linux-aws-headers-4.4.0-1041 linux-aws-headers-4.4.0-1043 linux-aws-headers-4.4.0-1044 linux-aws-headers-4.4.0-1047
  linux-aws-headers-4.4.0-1048 linux-aws-headers-4.4.0-1049 linux-aws-headers-4.4.0-1052 linux-aws-headers-4.4.0-1054
  linux-aws-headers-4.4.0-1057 linux-aws-headers-4.4.0-1060 linux-aws-headers-4.4.0-1061 linux-aws-headers-4.4.0-1062
  linux-aws-headers-4.4.0-1065 linux-headers-4.4.0-1022-aws linux-headers-4.4.0-1028-aws linux-headers-4.4.0-1030-aws
  linux-headers-4.4.0-1031-aws linux-headers-4.4.0-1032-aws linux-headers-4.4.0-1035-aws linux-headers-4.4.0-1038-aws
  linux-headers-4.4.0-1039-aws linux-headers-4.4.0-1041-aws linux-headers-4.4.0-1043-aws linux-headers-4.4.0-1044-aws
  linux-headers-4.4.0-1047-aws linux-headers-4.4.0-1048-aws linux-headers-4.4.0-1049-aws linux-headers-4.4.0-1052-aws
  linux-headers-4.4.0-1054-aws linux-headers-4.4.0-1057-aws linux-headers-4.4.0-1060-aws linux-headers-4.4.0-1061-aws
  linux-headers-4.4.0-1062-aws linux-headers-4.4.0-1065-aws linux-image-4.4.0-1022-aws linux-image-4.4.0-1028-aws
  linux-image-4.4.0-1030-aws linux-image-4.4.0-1031-aws linux-image-4.4.0-1032-aws linux-image-4.4.0-1035-aws
  linux-image-4.4.0-1038-aws linux-image-4.4.0-1039-aws linux-image-4.4.0-1041-aws linux-image-4.4.0-1043-aws
  linux-image-4.4.0-1044-aws linux-image-4.4.0-1047-aws linux-image-4.4.0-1048-aws linux-image-4.4.0-1049-aws
  linux-image-4.4.0-1052-aws linux-image-4.4.0-1054-aws linux-image-4.4.0-1057-aws linux-image-4.4.0-1060-aws
  linux-image-4.4.0-1061-aws linux-image-4.4.0-1062-aws linux-image-4.4.0-1065-aws
0 upgraded, 0 newly installed, 63 to remove and 115 not upgraded.
After this operation, 2,800 MB disk space will be freed.
```
